README.md

```diff
@@ -188,3 +188,6 @@ Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Con
 
 ## Contact
 You can reach the admins, maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or intereact with the community. Join Us!
+
+## LICENSE
```

Comment:
Hey @Subhangini I would suggest you to convert `## LICENSE` to `## License`  it is not necessary to be in all caps